### ISSUE TITLE
Not possible to have responseContainer = "list" AND @ApiResponse 

### ISSUE THREAD

**Issue by who** [kongchen/swagger-maven-plugin#163]
When I use `@ApiOperation#responseContainer` AND `@ApiResponse`, the output never renders array type correctly.

**RequestMapping method**

```
    @ApiOperation(value = "testing", response = User.class, responseContainer = "list")
    @RequestMapping(value = "/testing", method = RequestMethod.GET)
    @ApiResponses(value = {
        @ApiResponse(code = 200, message = "success")
    })
    public Object testing() {
        return new Object();
    }
```

**Output**
  
```
"paths" : {
    "/testing/testing" : {
      "get" : {
        "tags" : [ "example" ],
        "summary" : "testing",
        "description" : "",
        "operationId" : "testing",
        "responses" : {
          "200" : {
            "description" : "success"
          }
        }
      }
    }
  
```

The above is incorrect; the output should be:

```
"paths" : {
    "/testing/testing" : {
      "get" : {
        "tags" : [ "example" ],
        "summary" : "testing",
        "description" : "",
        "operationId" : "testing",
        "responses" : {
          "200" : {
            "description" : "success",
            "schema" : {
              "type" : "array",
              "items" : {
                "$ref" : "#/definitions/User"
              }
            }
          }
        }
      }
    }
  }
```

**Comment by ndhar-1** [comment]
has this issue been fixed in swagger-maven plugin version 3.0.1 ?

**Comment by who** [comment]
@ndhar-1 it is fixed in 3.0.1-SNAPSHOT.  I'm not sure when @kongchen plans on releasing 3.0.1 officially.

**Comment by kongchen** [comment]
Sorry guys, will do it now, sorry for the delay. I was too busy these days and even do not have time to have enough sleep:(
